pages/docs/styled-system/theming/theme-tools.mdx

```diff
@@ -0,0 +1,233 @@
+following props can be used as part of the signature:
+
+| Prop          | Type / Value                                                                                                   |
+| ------------- | -------------------------------------------------------------------------------------------------------------- |
+| `colorScheme` | string value of a theme token (ie. `blue` or `green`)                                                          |
+| `theme`       | The theme object                                                                                               |
+| `colorMode`   | `Dark` or `Light` depending on the current value in the ColorMode Provider                                     |
+| `orientation` | `horizontal` or `vertical`, defined as the direction the component is laid out. (Like setting `flexDirection`) |
+
+And any other props passed to the component you want to access!
+
+## Colors
+
+Below are the methods that can be used to access and alter the colors from the
+theme.
+
+| Name                 | Description                                                                                                                                                                                                                                                                                          | Function Call                                                         |
+| -------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------- |
+| `getColor`           | Retrieves a color hex value. Needs the theme and color name, and can take a fallback hex if the color name does not return a valid hex from the theme.                                                                                                                                               | `getColor(theme, color, fallback)`                                    |
+| `tone`               | Determines if the given color is relatively "light" or "dark". Requires the theme and a color                                                                                                                                                                                                        | `tone(color)(theme)`                                                  |
+| `isDark` / `isLight` | Utilizes the `tone` method to return a boolean whether a color is "Dark" or "Light".                                                                                                                                                                                                                 | `isDark(color)(theme)`                                                |

```

Comment:
All of the `color` methods from the `theme-tools` package are deprecated and will be removed: https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme-tools/src/color.ts#L40 maybe @segunadebayo or @TimKolberger can chime in here to let us know when exactly that'll happen. But regardless, since we've deprecated these there's no need to document them.